***publications:***  
1. Carreno BM, Magrini V, Becker-Hapak M, Kaabinejadian S, Hundal J, Petti AA, Ly A, Lie WR, Hildebrand WH, Mardis ER, Linette GP. Cancer immunotherapy. ***A dendritic cell vaccine increases the breadth and diversity of melanoma neoantigen-specific T cells***. Science. 2015 May 15;348(6236):803-8. doi: 10.1126/science.aaa3828. Epub 2015 Apr 2. PMID: 25837513; PMCID: PMC4549796.  

2. Snyder A, Makarov V, Merghoub T, Yuan J, Zaretsky JM, Desrichard A, Walsh LA, Postow MA, Wong P, Ho TS, Hollmann TJ, Bruggeman C, Kannan K, Li Y, Elipenahli C, Liu C, Harbison CT, Wang L, Ribas A, Wolchok JD, Chan TA. Genetic basis for clinical response to CTLA-4 blockade in melanoma. N Engl J Med. 2014 Dec 4;371(23):2189-2199. doi: 10.1056/NEJMoa1406498. Epub 2014 Nov 19. Erratum in: N Engl J Med. 2018 Nov 29;379(22):2185. PMID: 25409260; PMCID: PMC4315319. [link](https://www.nejm.org/doi/pdf/10.1056/NEJMoa1406498?articleTools=true)  

3. Smart, A., Margolis, C., Pimentel, H. et al. Intron retention is a source of neoepitopes in cancer. Nat Biotechnol 36, 1056–1058 (2018). https://doi.org/10.1038/nbt.4239  
[link](https://www.nature.com/articles/nbt.4239#Sec14)  

4. Broseus L, Ritchie W. Challenges in detecting and quantifying intron retention from next generation sequencing data. Comput Struct Biotechnol J. 2020 Feb 26;18:501-508. doi: 10.1016/j.csbj.2020.02.010. PMID: 32206209; PMCID: PMC7078297.  
[link](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7078297/)  

5. Applications of Next-Generation Sequencing in Neoantigen Prediction and Cancer Vaccine Development.Elizabeth M. Lancaster, David Jablons, and Johannes R. Kratz.Genetic Testing and Molecular Biomarkers.Feb 2020.59-66.http://doi.org/10.1089/gtmb.2018.0211



***tools for neoantigen prediction***

**MHCcombine**  
M. Bonsack, S. Hoppe, J. Winter, C. Zeller, M. D. Küpper, E. Schitter, R. Blatnik, A. B  
***Riemer Performance evaluation of MHC class-I binding prediction tools based on an experimentally validated MHC-peptide binding dataset.***  

**NetMHCpan 4.0**  
V. Jurtz, S. Paul, M. Andreatta,P. Marcatili, B. Peters, M. Nielsen
***NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data.***  
The Journal of Immunology (2017) PMID: 28978689    

M. Andreatta, M. Nielsen  
***NetMHCpan-3.0: improved prediction of binding to MHC class I molecules integrating information from multiple receptor and peptide length data sets.***  
Genome Medicine (2016) PMID: 27029192  

I. Hoof, B. Peters, J. Sidney, L. E. Pedersen, A. Sette, O. Lund, S. Buus, and M. Nielsen
***NetMHCpan, a method for MHC class I binding prediction beyond humans.***  
Immunogenetics (2009) PMID: 19002680


**NetMHC 4.0**  
M. Andreatta, M. Nielsen  
***Gapped sequence alignment using artificial neural networks: application to the MHC class I system.***  
Bioinformatics (2016) PMID: 26515819  

M. Nielsen, C. Lundegaard, P. Worning, S. L. Lauemøller, K. Lamberth, S. Buus, S. Brunak, and O. Lund   
***Reliable prediction of T-cell epitopes using neural networks with novel sequence representations.***  
Protein Science (2003) PMID: 12717023

**NetMHC 3.4**  
M. Nielsen, C. Lundegaard, P. Worning, S. L. Lauemøller, K. Lamberth, S. Buus, S. Brunak, and O. Lund  
***Reliable prediction of T-cell epitopes using neural networks with novel sequence representations.***  
Protein Science (2003) PMID: 12717023  

C. Lundegaard, K. Lamberth, M. Harndahl, S. Buus, O. Lund, and M. Nielsen  
***NetMHC-3.0: Accurate web accessible predictions of Human, Mouse, and Monkey MHC class I affinities for peptides of length 8-11.***  
Nucleic Acids Research - Web Server Issue (2008) PMID: 18463140  

***If peptide lengths other than 9mers were predicted please cite:***

C. Lundegaard, O. Lund, and M. Nielsen  
***Accurate approximation method for prediction of class I MHC affinities for peptides of length 8, 10 and 11 using prediction tools trained on 9mers.***  
Bioinformatics (2008) PMID: 18413329

**NetMHCpan 3.0**  
M. Andreatta, M. Nielsen  
***NetMHCpan-3.0: improved prediction of binding to MHC class I molecules integrating information from multiple receptor and peptide length data sets.***  
Genome Medicine (2016) PMID: 27029192


I. Hoof, B. Peters, J. Sidney, L. E. Pedersen, A. Sette, O. Lund, S. Buus, and M. Nielsen  
***NetMHCpan, a method for MHC class I binding prediction beyond humans.***  
Immunogenetics (2009) PMID: 19002680  

**NetMHCpan 2.8**    
I. Hoof, Peters B, J. Sidney, L. Pedersen, A. Sette, O. Lund, S. Buus, M. Nielsen  
***NetMHCpan, a method for MHC class I binding prediction beyond humans.***  
Immunogenetics (2007) PMID: 19002680  

M. Nielsen, C. Lundegaard, T. Blicher, K. Lamberth, M. Harndahl, S. Justesen, G. Roder, B. Peters, A. Sette, O. Lund, S. Buus  
***NetMHCpan, a method for quantitative predictions of peptide binding to any HLA-A and -B locus protein of known sequence.***  
PLoS One (2007) PMID: 17726526  

**NetMHCcons 1.1**  
E. Karosiene, C. Lundegaard, O. Lund, M. Nielsen  
***NetMHCcons: a consensus method for the major histocompatibility complex class I predictions.***  
Immunogenetics (2012) PMID: 22009319  

**PickPocket 1.1**  
H. Zhang, O. Lund, M. Nielsen  
***The PickPocket method for predicting binding specificities for receptors based on receptor pocket similarities: application to MHC-peptide binding.***  
Bioinformatics (2009) PMID: 19297351  

**IEDB Consensus**  
M. Moutaftsi, B. Peters, V. Pasquetto, D. Tscharke, J. Sidney, H. Bui, H. Grey, A. Sette  
***A consensus epitope prediction approach identifies the breadth of murine T(CD8+)-cell responses to vaccinia virus.***  
Nature Biotechnology (2006) PMID: 16767078  

**IEDB SMMPMBEC**  
Y. Kim, J. Sidney, C. Pinilla, A. Sette, B. Peters  
***Derivation of an amino acid similarity matrix for peptide:MHC binding and its application as a Bayesian prior.***  
BMC Bioinformatics (2009) PMID: 19948066  

**IEDB SMM**  
B. Peters, A. Sette  
***Generating quantitative models describing the sequence specificity of biological processes with the stabilized matrix method.***  
BMC Bioinformatics (2005) PMID: 15927070  

**SYFPEITHI**  
H. Rammensee, J. Bachmann, N. N. Emmerich, O. A. Bachor and S. Stevanovic  
***SYFPEITHI: database for MHC ligands and peptide motifs.***  
Immunogenetics (1999) PMID: 10602881  

**[RANKPEP](http://imed.med.ucm.es/Tools/rankpep.html)**  
RANKPEP that uses position specific scoring matrices (PSSMs) or profiles for the prediction of peptide-MHC class I (MHCI) binding as a basis for CD8 T-cell epitope identification.  
* Reche PA, Glutting JP and Reinherz EL Prediction of MHC Class I Binding Peptides Using Profile Motifs. Human Immunology 63, 701-709 (2002).  
* Reche PA, Glutting JP, Zhang H, Reinherz EL.Enhancement to the RANKPEP resource for the prediction of peptide binding to MHC molecules using profiles. 456:405-419 (2004)  
* Manoj Bhasin, Ellis L. Reinhez and Pedro A. Reche. Modeling features of immunodominance into T-cell epitope identification. 2nd International Immunoinformatics Symposium, Boston University, March 7-9, 2005.


***References about HLA supertypes***

A. Sette, and J. Sidney  
***Nine major HLA class I supertypes account for the vast preponderance of HLA-A and -B polymorphism.***  
Immunogenetics (1999) PMID: 10602880  

O. Lund, M. Nielsen, C. Kesmir, A. G. Petersen, C. Lundegaard, P. Worning, C. Sylvester-Hvid, K. Lamberth, G. Roder, S. Justesen, S. Buus, and S. Brunak  
***Definition of supertypes for HLA molecules using clustering of specificity matrices.***  
Immunogenetics (2004) PMID: 14963618  

J. Sidney, B. Peters, N. Frahm, C. Brander, and A. Sette  
***HLA class I supertypes: a revised and updated classification.***  
BMC Immunology (2008) PMID: 18211710  