1. Cui, C, Tian, X, Wu, J, et al. T cell receptor β‐chain repertoire analysis of tumor‐infiltrating lymphocytes in pancreatic cancer. Cancer Sci. 2019; 110: 61– 71. https://doi.org/10.1111/cas.13877

2. Chen, Z., Zhang, C., Pan, Y., Xu, R., Xu, C., Chen, Z., Lu, Z. and Ke, Y. (2016), T cell receptor β‐chain repertoire analysis reveals intratumour heterogeneity of tumour‐infiltrating lymphocytes in oesophageal squamous cell carcinoma. J. Pathol., 239: 450-458. doi:10.1002/path.4742  

3. Gerlinger M, Quezada SA, Peggs KS, Furness AJ, Fisher R, Marafioti T, Shende VH, McGranahan N, Rowan AJ, Hazell S, Hamm D, Robins HS, Pickering L, Gore M, Nicol DL, Larkin J, Swanton C. Ultra-deep T cell receptor sequencing reveals the complexity and intratumour heterogeneity of T cell clones in renal cell carcinomas. J Pathol. 2013 Dec;231(4):424-32. doi: 10.1002/path.4284. PMID: 24122851; PMCID: PMC4241038.

4. Mehdi Yousfi Monod, Véronique Giudicelli, Denys Chaume, Marie-Paule Lefranc, IMGT/JunctionAnalysis: the first tool for the analysis of the immunoglobulin and T cell receptor complex V–J and V–D–J JUNCTIONs, Bioinformatics, Volume 20, Issue suppl_1, 4 August 2004, Pages i379–i385, https://doi.org/10.1093/bioinformatics/bth945

5. James Robinson, Dominic J Barker, Xenia Georgiou, Michael A Cooper, Paul Flicek, Steven G E Marsh, IPD-IMGT/HLA Database, Nucleic Acids Research, Volume 48, Issue D1, 08 January 2020, Pages D948–D955, https://doi.org/10.1093/nar/gkz950

6. Daniel J Rigden, Xosé M Fernández, The 27th annual Nucleic Acids Research database issue and molecular biology database collection, Nucleic Acids Research, Volume 48, Issue D1, 08 January 2020, Pages D1–D8, https://doi.org/10.1093/nar/gkz1161 

7. Shugay M, Bagaev DV, Turchaninova MA, et al. VDJtools: Unifying Post-analysis of T Cell Receptor Repertoires. PLoS Comput Biol. 2015;11(11):e1004503. Published 2015 Nov 25. doi:10.1371/journal.pcbi.1004503

8. Attaf, M., Huseby, E. & Sewell, A. αβ T cell receptors as predictors of health and disease. Cell Mol Immunol 12, 391–399 (2015). https://doi.org/10.1038/cmi.2014.134

9. Attaf, M., Huseby, E. & Sewell, A. αβ T cell receptors as predictors of health and disease. Cell Mol Immunol 12, 391–399 (2015). https://doi.org/10.1038/cmi.2014.134

10. Immune system. https://www.britannica.com/print/article/283636  

11. [TCR3d](https://tcr3d.ibbr.umd.edu/tcr_com),T cell receptor structural repertoire database  

12. [SAbPred](http://opig.stats.ox.ac.uk/webapps/newsabdab/sabpred/),The Antibody Prediction Toolbox   

13. [STCRDab](http://opig.stats.ox.ac.uk/webapps/stcrdab/),the structural T-cell receptor database,is an online resource that automatically collects and curates TCR structural data from the Protein Data Bank. For each entry, the database provides annotations, such as the α/β or γ/δ chain pairings, major histocompatibility complex details, and where available, antigen binding affinities. In addition, the orientation between the variable domains and the canonical forms of the complementarity-determining region loops are also provided. Users can select, view, and download individual or bulk sets of structures based on these criteria. Where available, STCRDab also finds antibody structures that are similar to TCRs, helping users explore the relationship between TCRs and antibodies.   

14. Bouquet J, Gardy JL, Brown S, Pfeil J, Miller RR, Morshed M, Avina-Zubieta A, Shojania K, McCabe M, Parker S, Uyaguari M, Federman S, Tang P, Steiner T, Otterstater M, Holt R, Moore R, Chiu CY, Patrick DM; Complex Chronic Disease Study Group. RNA-Seq Analysis of Gene Expression, Viral Pathogen, and B-Cell/T-Cell Receptor Signatures in Complex Chronic Disease. Clin Infect Dis. 2017 Feb 15;64(4):476-481. doi: 10.1093/cid/ciw767. PMID: 28172519; PMCID: [PMC5850034](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850034/).  

15. [powerTCR](https://www.bioconductor.org/packages/release/bioc/html/powerTCR.html),his package provides a model for the clone size distribution of the TCR repertoire. Further, it permits comparative analysis of TCR repertoire libraries based on theoretical model fits.  


